### ISSUE TITLE
[Enhancement] Add YIFY(yts.to) support

### ISSUE THREAD

**Issue by OmgImAlexis** [Stephen304/bitcannon#84]
I contacted them and they support these two end points. It'd be nice to add these to the defaults since they're used by a LOT of people.

https://yts.to/tz_hourly.txt.gz
https://yts.to/tz_daily.txt.gz

**Comment by alexandre-p** [comment]
Great idea.
I tried to had it to my config but unfortunately it failed. When trying to get the file, Go returns the following error : `Get https://yts.to/tz_daily.txt.gz: x509: certificate signed by unknown authority (possibly because of "x509: cannot verify signature: algorithm unimplemented" while trying to verify candidate authority certificate "COMODO RSA Certification Authority")`

Thanks to http://bridge.grumpy-troll.org/2014/05/golang-tls-comodo/ I found the solution.
You simply need to add `_ "crypto/sha512"` to the imports in api/import.go

I'll try to find some time later today to do a proper pull request.
